iot_config/src/org.rs

```diff
@@ -276,6 +462,8 @@ pub enum OrgStoreError {
     DecodeKey(#[from] helium_crypto::Error),
     #[error("Route Id parse error: {0}")]
     RouteIdParse(#[from] sqlx::types::uuid::Error),
+    #[error("Invalid or Unauthorized update: {0}")]
+    InvalidUpdate(String),
```

Comment:
There's enough ways to fail an update it might make sense to differentiate invalid vs unauthorized.
Especially since unauthorized checks fall through to a generic message.